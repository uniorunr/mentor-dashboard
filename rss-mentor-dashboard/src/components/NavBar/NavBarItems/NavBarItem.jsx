import React from 'react';
import PropTypes from 'prop-types';
import './NavBarItem.css';

const NavBarItem = ({ link, name }) => (
  <li>
    <a href={link}>{name}</a>
  </li>
);

NavBarItem.propTypes = {
  link: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired,
};

export default NavBarItem;
