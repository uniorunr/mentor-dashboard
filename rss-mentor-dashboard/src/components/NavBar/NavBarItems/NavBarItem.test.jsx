import React from 'react';
import renderer from 'react-test-renderer';
import NavBarItem from './NavBarItem';

it('renders correctly', () => {
  const tree = renderer
    .create(<NavBarItem link="/" name="Home" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
