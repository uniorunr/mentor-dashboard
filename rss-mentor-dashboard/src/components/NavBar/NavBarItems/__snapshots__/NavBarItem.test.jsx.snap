// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li>
  <a
    href="/"
  >
    Home
  </a>
</li>
`;
